package.json

```diff
@@ -49,7 +49,7 @@
     "test": "npm run test:main && npm run test:fp",
     "test:fp": "node test/test-fp",
     "test:main": "node test/test",
-    "validate": "npm run style & npm run test"
+    "validate": "npm run style & npm run test & wait"
   },
```

Comment:
Hmm... it looks like opt-cli's command `opt` is interfering with another command with the same name, thus the rendered output in the logs (line 184+). I will have a more thorough look soon.